contract/src/api.rs

```diff
@@ -125,16 +125,19 @@ impl Contract {
     ///     }
     ///   }
     /// })
+    ///
+    /// When `refund_unused_deposit` is set to `true`, the part of the deposit that covers unused
+    /// storage will be refunded to the caller.
+    ///
     /// ```
     #[payable]
-    pub fn set(&mut self, mut data: Value) {
+    pub fn set(&mut self, mut data: Value, refund_unused_deposit: Option<bool>) {
```

Comment:
Similarly to `get` and `keys` methods I would introduce `options` parameter just to follow the style: https://github.com/NearSocial/social-db/blob/39b26f6904619e5b9d12de406f5ab5f9d66f7b8a/contract/src/api.rs#L55

Reply:
Thanks, fixed in 5d751a8e3d4413451d893f88a764aead7b595f87